dnf-behave-tests/features/install-xml-base.feature

```diff
@@ -0,0 +1,147 @@
+Feature: Install packages with base:xml set
+
+
+Scenario: Install packages from local repository with local xml:base
+Given I copy directory "{context.dnf.repos_location}/dnf-ci-fedora" to "/temp-repo/dnf-ci-fedora"
+  And I create and substitute file "/etc/yum.repos.d/test.repo" with
+  """
+  [testrepo]
+  name=testrepo
+  baseurl=file://{context.dnf.installroot}/temp-repo
+  enabled=1
+  gpgcheck=0
+  """
+  And I do not set reposdir
+  And I use the repository "testrepo"
+  And I execute bash with args "createrepo_c --baseurl file://{context.dnf.installroot}/temp-repo/xml_base ." in directory "{context.dnf.installroot}/temp-repo"
+ Then the exit code is 0
+ Then I copy directory "{context.dnf.installroot}/temp-repo/dnf-ci-fedora" to "/temp-repo/xml_base/dnf-ci-fedora"
```

Comment:
This is quite confusing... One, I can't easily tell, but all these steps look to be the test setup, so they should be `Given`, right?

Also, these tests have a really long setup, I haven't figured out all the things that are going on here, but I'm wondering if some better step definition would help with this...

Reply:
Yes I agree, I will try to abstract several of the steps into new step with hopefully more descriptive name.